lib/sign_in/view/sign_in_page.dart

```diff
@@ -221,7 +221,10 @@ class _SignInBodyState extends State<SignInBody>
                                     ),
                                     TextButton(
                                       onPressed: () {
-                                        context.router.push(SignUpRoute());
```

Comment:
Please revert this changes, It seems like you forget to do so when testing

Reply:
Oh yeah

Thank you